File: frontend/src/app/dora/types/pipeline.ts
export interface Pipeline {
  name: string;
  id: string;
  orgId?: string;
  orgName?: string;
  repository: string;
  steps: string[];
}

export interface Org {
  orgName: string;
  orgId: string;
}

export interface PipelineEqual {
  id: string;
  step: string;
}

File: backend/src/main/java/heartbeat/controller/pipeline/dto/response/Pipeline.java
package heartbeat.controller.pipeline.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Pipeline {

	String id;

	String name;

	String orgId;

	String orgName;

	String repository;

	List<String> steps;

}
